### PR TITLE
Take over Squirrel

### PR DESCRIPTION
**Pull request by Enduriel**
The [original author](https://github.com/iAmRoland/Sublime-Squirrel) of the squirrel plugin is unresponsive (hasn't replied to my email for months and 0 activity on any of their easily viewable social media for years). I've worked on significantly improving the package since I started working with Squirrel and think it should be used as the default for Package Control.

Hope I'm doing this right, used [this](https://github.com/wbond/package_control_channel/pull/5346) as a template.

### CODE REVIEW

**Comment by Ultra-Instinct-05** [comment]
@iAmRoland Are you okay with @Enduriel taking over the maintainer ship of the package ? If so, please respond within 14 days.

**Comment by iAmRoland** [comment]
Sorry, was supposed to get back but completely forgot.

Yeah you can go ahead and transfer the maintainer ship. The changes in your branch seem pretty neat, if I ever use Squirrel again I'll try it out 😎

**Comment by braver** [comment]
Thanks. 

Please don't ship color schemes with your syntax. There are hundreds of color schemes out there and people really want to use what they like. And people like to use their editor for loads of different languages. If you stick to the scope naming guidelines there is usually no need to tweak color schemes, and it allows color schemes author to effectively optimize for a wide range of languages.

**Comment by Enduriel** [comment]
@iAmRoland awesome ty! I genuinely thought something had happened to you since you were inactive on every social media you had linked so glad to hear you're okay.

@braver that is fair, in theory that should be the case, for the two specific themes I included it unfortunately was decidedly *not* so I bundled them together for the people I was sharing it with but I'll move it to a different repo.

**Comment by Enduriel** [comment]
Done

**Comment by Ultra-Instinct-05** [comment]
I forgot to mention another thing. Since your package is using the `.sublime-syntax` format, you need to specify `"sublime_text": ">=3092"` since the format was introduced in that build. To continue for any ST2 users (for whatever goddamn reason), you can do something like this.

```json
"releases": [
    {
        "base": "https://github.com/iAmRoland/Sublime-Squirrel",
        "sublime_text": "<3092",
        "branch": "master"
    },
    {
        "sublime_text": ">=3092",
        "tags": true
    }
]
```

**Comment by jrappen** [comment]
Also, please add a license.

**Comment by Enduriel** [comment]
@Ultra-Instinct-05 done and @jrappen done.

Also I would argue if someone is still using ST2 they deserve anything that happens to them :P

File: repository/s.json [hunk 1]
```diff
@@ -3469,12 +3469,17 @@
...
-			"author": ["micheg", "Roland Piirsoo"],
+			"details": "https://github.com/Enduriel/Sublime-Squirrel",
+			"author": ["Enduriel", "micheg", "Roland Piirsoo"],
 			"labels": ["language", "syntax", "snippets", "completions", "squirrel"],
 			"releases": [
 				{
-					"sublime_text": "*",
+					"base": "https://github.com/iAmRoland/Sublime-Squirrel",
+					"sublime_text": "<3092",
+					"branch": "master"
```
Comment (braver) [inline repository/s.json:3479]: Actually, ST < 3092 is not supported either, there is no need to take those versions into account here. Especially not via branch based releases.

**Review by braver — CHANGES_REQUESTED** [review]

**Comment by braver** [comment]
@Ultra-Instinct-05 is strictly correct, but we don't enforce support for < 3092. "ST3 support" is assumed to mean versions newer than that.

**Comment by jrappen** [comment]
If you ask me, you're wasting your time with ST3 support. I believe the vast majority has already moved on.

I'd target `>=4107` (first public stable ST4 Build).

**Comment by braver** [comment]
Honestly, after a package is submitted nobody verifies the compatibility anymore. Make life easy for yourself and just support whatever you’re using yourself. Nobody using an ancient unsupported version of Sublime Text is looking for new packages.

**Comment by Enduriel** [comment]
I think from what I understand the thing I'd most prefer to go with would be @jrappen's suggestion, as if I use features that have been added later than that (which I might end up doing at some point but it'd definitely be a conscious decision) I can cap the release for those versions to a specific tag from what I understand?

And like I don't know, someone might have a rando reason to not use the latest latest version of ST4 which in my opinion is fair enough. If they're using ST4 it will be fine (the sublime-syntax file was first done for my own personal stuff before ST4 even released).

Is that alright?

**Comment by braver** [comment]
I would say just drop the release for < 4107 and if someone has some random reason to use an outdated version of ST you can tell in them (in the readme) how to manually install a specific version of your package. You don't need to do everything via package control, just make life easy for yourself.

**Comment by Enduriel** [comment]
@braver I mean it doesn't cost me anything to point older versions to the other repo just in case, not sure why not to do that?

**Comment by braver** [comment]
Sure, whatever you like. I just want to get rid branch thing.

**Comment by braver** [comment]
Please respond to the feedback to continue this PR.

**Comment by Enduriel** [comment]
Sorry for taking so long, @braver I've removed the section like you asked.

**Pull request merged**